test/com/google/enterprise/adaptor/sharepoint/HtmlResponseWriterTest.java

```diff
@@ -19,6 +19,8 @@
 import com.google.enterprise.adaptor.Config;
 import com.google.enterprise.adaptor.DocId;
 import com.google.enterprise.adaptor.DocIdPusher;
+import com.google.enterprise.adaptor.testing.RecordingDocIdPusher;
+import com.google.enterprise.adaptor.testing.UnsupportedDocIdPusher;
```

Comment:
UnsupportedDocIdPusher is unused.

Reply:
Replaced local copy of UnsupportedDocIdPusher with library UnsupportedDocIdPusher. 
DocIdPusher object is the instance of UnsupportedDocIdPusher.